src/test/kotlin/Sum.kt

```diff
@@ -0,0 +1,43 @@
+@file:Suppress("unused")
+
+/**
+ * This functions sums two integer numbers.
+ *  >>> assertEquals(11, sum(5, 6))
+ */
+fun sum(a: Int, b: Int) = a + b
```

Comment:
I should give it a name that does not clashes with standard library (maybe add some suffix - this name does not have to be very readable).